packages/eslint-plugin/lib/config/node.js

```diff
@@ -10,5 +10,6 @@ module.exports = {
   rules: merge(require('./rules/node'), {
     'no-process-env': 'off',
     'no-console': 'off',
+    'no-useless-token-range': 'off',
```

Comment:
We are turning this off as stuff will break? This will make rules less performant...
https://github.com/not-an-aardvark/eslint-plugin-eslint-plugin/blob/master/docs/rules/no-useless-token-range.md

Reply:
I forgot I turned this off. Nice catch.